mobify/dustjs
# Chain inheritance

Changes behaviour of block overriding when doing chain inheritence

Status: **Opened for visibility**

Reviewers: @noahadams @scalvert 
JIRA: https://mobify.atlassian.net/browse/GOLD-209
## Changes
- Re-introduces _SUPER_
- Brings over behaviour used in Mobify Client that allows you to override blocks way up the inheritance chain.
## How to test-drive this PR
- Include this dust build inside of an adaptive.js project and ensure that you can do block overriding from multiple levels up.
## Notes
- Some interesting discussions on this topic for further reading:
  https://github.com/linkedin/dustjs/issues/101
  https://github.com/linkedin/dustjs/pull/314
